tools/ci_build/build.py

```diff
@@ -12,6 +12,7 @@
 import subprocess
 import sys
 import hashlib
+from rewriter import rewrite_provider
 
```

Comment:
nit: 'rewriter' is a really vague name. can we name it something that makes it a little clearer that it is disabling ops that are not used? that it does that via rewriting the provider is more of a current implementation detail which could change.

Reply:
Sure, suggestions?

Reply:
Suggestions for better name?

Reply:
Switching to provider_rewriter